### PR TITLE
Set webhook endpoint to random uuid based string if not set

### PR DESCRIPTION
**Pull request by dpires**
Instead of returning null, which breaks certain Jenkins versions set the webhook endpoint to a random string.

### CODE REVIEW

File: src/main/java/jenkins/plugins/slack/webhook/WebhookEndpoint.java [hunk 1]
```diff
@@ -49,7 +50,7 @@ public WebhookEndpoint() {
     public String getUrlName() {
         String url = globalConfig.getSlackOutgoingWebhookURL();
         if (url == null || url.equals(""))
-            return null;
+            return UUID.randomUUID().toString().replaceAll("-", "");
```
Comment (yoanisgil) [inline src/main/java/jenkins/plugins/slack/webhook/WebhookEndpoint.java]: You should probably add a test to make sure it never returns null ;)

**Comment by adepue** [comment]
:+1: Thanks for fixing

**Comment by RVoigtHearsay** [comment]
:+1: Thanks for this

File: src/main/java/jenkins/plugins/slack/webhook/WebhookEndpoint.java (hunk 1 above)
Comment (dpires) [inline src/main/java/jenkins/plugins/slack/webhook/WebhookEndpoint.java]: Updated pr with a null/empty webhook url test case.

**Comment by samrocketman** [comment]
Looks OK to me +2 I'll let @kmadel review and merge.

**Comment by yoanisgil** [comment]
Thanks for fixing this so quickly.
